easybuild/easyconfigs/t/Tk/Tk-8.6.4-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.4'
+
```

Comment:
the versionsuffix you dropped fixes the issue with Tk not being found based on Tcl install prefix?

so, why is the patch being included here not working? is it incomplete?